apps/greenhouse/src/hooks/useApi.js

```diff
@@ -47,7 +47,9 @@ const useApi = () => {
           // temporary fix to forward initialFilters to the Plugins until middleware is implemented
           const appProps = { username: authData?.parsed?.fullName }
           appProps.initialFilters = {
-            support_group: authData?.parsed?.supportGroups?.map((group) => group),
+            ...(Array.isArray(authData?.parsed?.supportGroups) && {
+              support_group: authData.parsed.supportGroups.map((group) => group),
```

Comment:
Please don't use map here, If authData.parsed.supportGroups is already an array, .map((group) => group) is redundant and doesn’t add value—it simply returns the same array.